app/services/windows.ts

```diff
@@ -104,7 +104,28 @@ export function getComponents() {
   };
 }
 
-export interface IWindowOptions {
+export interface IElectronWindowOptions {
```

Comment:
You should be able to use `Electron.BrowserWindowConstructorOptions` instead of defining this yourself.